src/ai/defaultai_warfare.cc

```diff
@@ -1122,6 +1136,11 @@ BuildingNecessity DefaultAI::check_building_necessity(BuildingObserver& bo, cons
 	                         3)};
 	const uint16_t total_score = scores[0] + scores[1] + scores[2];
 
+	// help variable to determine wood availability in the economy
+	const int32_t stocked_wood_level = calculate_stocklevel(tribe_->safe_ware_index("log")) -
```

Comment:
this is needed to replace the usage of the old only log based wood policy but will result in the same logical values